roles/openshift_logging/templates/kibana.j2

```diff
@@ -32,6 +32,16 @@ spec:
           name: "kibana"
           image: {{image}}
           imagePullPolicy: Always
+{% if kibana_memory_limit is not none or kibana_cpu_limit is not none %}
```

Comment:
passed in from the template.  due you think we need these to make the the defined check too?